CHANGELOG.md

```diff
@@ -6,6 +6,51 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [6.1.0] - 2021-03-10
+
+### Added
+
+- Add Base Controller v2 ([#358]https://github.com/MetaMask/controllers/pull/358)
```

Comment:
All of these links are missing parentheses

```suggestion
- Add Base Controller v2 ([#358](https://github.com/MetaMask/controllers/pull/358))
```

Check out the rich diff to see what I mean

Reply:
😢  going to update it!

Reply:
updated!